modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select.soy

```diff
@@ -115,15 +115,25 @@
 
 				{if not $readOnly}
 					<div class="drop-chosen {$open ? '' : 'hide'}">
-						<div class="search-chosen">
-							<div class="select-search-container">
-								{if $selectSearchIcon}
-									<a class="" href="javascript:;">{$selectSearchIcon}</a>
-								{/if}
-							</div>
-							<input autocomplete="off" class="drop-chosen-search" placeholder="{$strings.search}" type="text">
+						<div class="dropdown-menu" aria-labelledby="theDropdownToggleId">
+							<forms>
```

Comment:
@alinedoleron Is this `<forms>` tag correct?

Reply:
OMG! Thank you, @brunobasto! It should be <form>.